plugins/modules/cloud/opennebula/one_vm.py

```diff
@@ -1586,7 +1612,8 @@ def main():
             'absent': wait_for_done,
             'rebooted': wait_for_running,
             'poweredoff': wait_for_poweroff,
-            'running': wait_for_running
+            'running': wait_for_running,
+            'release': wait_for_running
```

Comment:
```suggestion
            'release': wait_for_running,
```
With that the next addition will only add a line, not modify this line as well, and so will have a cleaner diff.